cmd/tikv-ctl/src/cmd.rs

```diff
@@ -107,7 +107,7 @@ pub enum Cmd {
         #[structopt(subcommand)]
         cmd: RaftCmd,
     },
-    /// Print region size
+    /// Print region size and key count
     Size {
```

Comment:
the command is called `size`, but it actually prints size and key count

Reply:
How about adding a new command? I don't want to rename an existing command.

Reply:
+1